docs/pages/guides/spend-permissions/quick-start.mdx

```diff
@@ -0,0 +1,702 @@
+import { Callout } from "vocs/components";
+
+# Getting Started with Spend Permissions
+
+This guide will walk you through a simple example of building an app that leverages Spend Permissions using 
+[OnchainKit](https://onchainkit.xyz/), [Viem](https://viem.sh/) and [Wagmi](https://wagmi.sh/).
+
+A complete example of this demo can be found [here](https://github.com/ilikesymmetry/subscribe-onchain).  
+Smart contract deployment addresses for `SpendPermissionManager.sol` can be found [here](https://github.com/coinbase/spend-permissions).
+
+:::steps
+### Set up a basic app template using OnchainKit
+Set up a boilerplate React/Next app by running the following command and following the instructions:
+
+```bash
+npm create onchain
+```
+
+This will generate an app that is ready to run and contains a wallet connection button that users can use
+to connect their smart wallet to the application.
+
+From here, we'll modify the app to assemble, sign, approve and use a spend permission to spend our users' funds!
+
+
+### Install additional dependencies
+
+We'll need a few additional dependencies: [Viem](https://viem.sh/), [Wagmi](https://wagmi.sh/), and [TanStack](https://tanstack.com/query/latest)
+
+Run the following command to install these with our package manager, [Bun](https://bun.sh/)
+
+```bash
+bun add viem wagmi @tanstack/react-query
+```
+
+### Set up your spender wallet
+
+<Callout type="danger">
+Always secure your private keys appropriately!  
+We insecurely use an environment variable in this demo for simplicity.
+</Callout>
+
+Create a local `.env` file and add two environment variables:
+
+```
+SUBSCRIPTION_PRIVATE_KEY=
+NEXT_PUBLIC_SUBSCRIPTION_SPENDER=
+```
+
+These will represent our spender private key and the address of our spender smart contract wallet.
+
+You can generate a random private key [here](https://privatekeys.pw/keys/ethereum/random), or maybe you have a development private key
```

Comment:
maybe just recommend viem or foundry

Reply:
in https://github.com/coinbase/paymaster-bundler-examples we recommend `cast wallet new`